libs/shared/ui/src/lib/date-picker-custom-header/date-picker-custom-header.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { DatePickerCustomHeaderComponent } from './date-picker-custom-header.component';
+
+describe('DatePickerCustomHeaderComponent', () => {
+  let component: DatePickerCustomHeaderComponent;
```

Comment:
I think you need the generics here as well:
DatePickerCustomHeaderComponent<any>